lib/browser/api/web-contents.ts

```diff
@@ -462,8 +462,7 @@ const addReturnValueToEvent = (event: any) => {
 };
 
 const loggingEnabled = () => {
-  return Object.prototype.hasOwnProperty.call(process.env, 'ELECTRON_ENABLE_LOGGING') ||
-    process.argv.some(arg => arg.toLowerCase().startsWith('--enable-logging'));
+  return process.env.ELECTRON_ENABLE_LOGGING || app.commandLine.hasSwitch('enable-logging');
```

Comment:
teeeeechnically the logic for this in `base::Environment` is different. Ideally we'd just query this info directly from the C++ which actually controls logging, rather than trying to reimplement this.

Things that enable logging in the C++ without tripping this test, for example:
```
ELECTRON_ENABLE_LOGGING=0
electron_enable_logging=1

# only on non-Windows platforms:
ELECTRON_ENABLE_LOGGING=
```

I don't think it's very important to match exactly, but JFYI.